static/js/public/snap-details/map.js

```diff
@@ -81,31 +95,38 @@ export default function renderMap(el, snapData) {
 
         if (countrySnapData) {
           tooltip
-            .style('top', pos[1] + 'px')
-            .style('left', pos[0] + 'px')
-            .style('display', 'block');
-
-          let content = ['<span class="u-no-margin--top">', countrySnapData.name];
-          if (countrySnapData['number_of_users'] !== undefined) {
-            content.push(`<br />${countrySnapData['number_of_users']} active`);
+            .style("top", pos[1] + "px")
+            .style("left", pos[0] + "px")
+            .style("display", "block");
+
+          let content = [
+            '<span class="u-no-margin--top">',
+            countrySnapData.name
+          ];
+          if (countrySnapData["number_of_users"] !== undefined) {
+            content.push(`<br />${countrySnapData["number_of_users"]} active`);
           }
-          content.push('</span>');
+          content.push("</span>");
           tooltipMsg.html(
             `<span
                class="snapcraft-territories__swatch"
-               style="background-color: rgb(${countrySnapData.color_rgb[0]}, ${countrySnapData.color_rgb[1]}, ${countrySnapData.color_rgb[2]})"></span>
-             ${content.join(' ')}`
+               style="background-color: rgb(${countrySnapData.color_rgb[0]}, ${
```

Comment:
Prettier seems to do weird things to long template literals. It breaks the line inside `${...}` to keep in 80 chars, indents next line in weird way making ESLint complain about indentation.